NSS-Day-Cohort-68/rare-client-rare-team-1
# Categories lmw

Closes https://github.com/NSS-Day-Cohort-68/rare-client-rare-team-1/issues/38

The purpose of this PR is to allow users to see all category labels when navigating to the /category-management tab or when selecting Category Management from the NavBar.


Changes made:

Added Categories component to get and display the categories from the API.

Steps to test:

On server side (rare-api) start the debugger
Then on the client side, navigate to the Category Management page and verify categories are displayed in alphabetical order.
Click on the Create Category button.
Input a category label into the text box and press "Save" button
Page should navigate back to the Category Management page.

Check if test category is displayed and in alphabetical order.
Check if the tag was added to db.sqlite3
